src/components/EntityManagerApp/SingleEntityView/index.tsx

```diff
@@ -111,6 +147,12 @@ type Props = {
   entity: Entity;
 };
 
+type FieldValueUI = {
```

Comment:
We could name this `FieldValueMetadata` or `FieldValueUIMetadata`. Ending in `UI` isn't accurate since this is a type, not a user interface.